### PR TITLE
Update editor to rdfa-backwards-compatibility build

### PR DESCRIPTION
**Pull request by elpoelma**
### Overview
This PR updates the editor to the rdfa-backwards-compatible build (https://github.com/lblod/ember-rdfa-editor/pull/1155)
This build includes some API changes:
- `rdfaAttrSpec` is now a function
- usage of the `...WithConfig` configurable nodespecs

This PR also includes some changes to the citation plugin to support the `rdfaAware` system.

##### connected issues and PRs:
https://github.com/lblod/ember-rdfa-editor/pull/1155

### How to test/reproduce
- `pnpm start`
- Visit the different dummy pages, all node-specs should be in `rdfaAware` mode and the rdfa-features should work as expected

### Challenges/uncertainties
It does not seem very useful to also convert all the nodespecs provided by this package to be backwards compatible.

### Checks PR readiness
- [ ] UI: works on smaller screen sizes
- [ ] UI: feedback for any loading/error states
- [ ] Check if dummy app is correctly updated
- [ ] Check cancel/go-back flows
- [ ] changelog
- [ ] npm lint
- [ ] no new deprecations

### CODE REVIEW

File: tests/utils/editor.ts [hunk 1]
```diff
@@ -125,7 +125,7 @@ export const SAMPLE_SCHEMA = new Schema({
           )[0] as HTMLElement,
       },
     ]),
-    invisible_rdfa,
+    invisible_rdfa: inlineRdfaWithConfig({ rdfaAware: true }),
```
Comment (piemonkey) [inline tests/utils/editor.ts:128]: Typo: `invisibleRdfaWithConfig`

**Review by piemonkey — COMMENTED** [review]
When trying to insert anything from the sidebar (in the index route), I get an error:

```
Uncaught TypeError: can't access property "indexOf", tagName is undefined
```

**Comment by elpoelma** [comment]
> When trying to insert anything from the sidebar (in the index route), I get an error:
> 
> ```
> Uncaught TypeError: can't access property "indexOf", tagName is undefined
> ```

Interesting, will check if I can reproduce that.

File: tests/utils/editor.ts (hunk 1 above)
Comment (elpoelma) [reply tests/utils/editor.ts]: Thanks!

**Comment by elpoelma** [comment]
> When trying to insert anything from the sidebar (in the index route), I get an error:
> 
> ```
> Uncaught TypeError: can't access property "indexOf", tagName is undefined
> ```

Does it occur with a specific action?

**Comment by piemonkey** [comment]
> > When trying to insert anything from the sidebar (in the index route), I get an error:
> > ```
> > Uncaught TypeError: can't access property "indexOf", tagName is undefined
> > ```
> 
> Does it occur with a specific action?

It happens in the `insertStructure` command when trying to use any of the article structures in the `insert` menu in the sidebar. Actually, it's perhaps just `insert article` but once it has happened once it happens with anything from then on.

**Comment by elpoelma** [comment]
> > > When trying to insert anything from the sidebar (in the index route), I get an error:
> > > ```
> > > Uncaught TypeError: can't access property "indexOf", tagName is undefined
> > > ```
> > 
> > 
> > Does it occur with a specific action?
> 
> It happens in the `insertStructure` command when trying to use any of the article structures in the `insert` menu in the sidebar. Actually, it's perhaps just `insert article` but once it has happened once it happens with anything from then on.

That specific issue should be fixed now, it was an issue in the `toDOM` function of the `besluit_article_header` nodespec.
I did notice that the current implementation of a decision article is far from usable, so we should probably fix it before releasing the plugins/going to embeddable.

**Pull request merged**